
#won't do anything until ready
bool get_ready

#choice stage 
bool start_stage1
bool start_stage2

#start joy control
bool joy_available

